app/controllers/cohort_lists_controller.rb

```diff
@@ -6,7 +6,9 @@ def new
   end
 
   def create
-    @cohort_list = CohortList.new(cohort_list_params)
+    @cohort_list = CohortList.new(
+      cohort_list_params.merge(team: current_user.team)
+    )
```

Comment:
If you rebase this then it shouldn't be necessary anymore, we're `.merge`ing team in the `cohort_list_params` method below on `main`